source/partials/_site_footer.html.erb

```diff
@@ -31,7 +31,7 @@
       { name: 'Sharesight Blog', href: unlocalized_url('blog'), title: 'Read the Sharesight Blog' },
       { name: 'Webinars & Events', page: 'events' },
       { name: 'Privacy Policy', page: 'privacy-policy' },
-      { name: 'Terms of Use', page: 'sharesight-terms-of-use' },
+      { name: 'Terms of Use', href: localize_url('sharesight-terms-of-use', locale_id: locale_obj[:id]), title: "Terms of Use | #{locale_obj[:append_title]}" },
```

Comment:
Ah…!  That's a good thing to remember, we will have to hardcode these titles now.  I think you did this right.